src/components/voucherHeader/voucherHeader.css

```diff
@@ -0,0 +1,20 @@
+* {
```

Comment:
this style is global, this is not a part of the voucher component, so, you can move to your global file

Reply:
truth, I added in global.css